src/app/Fake.Core.Target/Target.fs

```diff
@@ -834,22 +840,36 @@ module Target =
                 if not allowArgs && arguments <> [] then
                     failwithf "The following arguments could not be parsed: %A\nTo forward arguments to your targets you need to use \nTarget.runOrDefaultWithArguments instead of Target.runOrDefault" arguments
                 match target with
-                | Some t -> runInternal singleTarget parallelJobs t arguments |> ignore
-                | None -> fDefault singleTarget parallelJobs arguments
+                | Some t -> Some(fun () -> runInternal singleTarget parallelJobs t arguments)
+                | None when defaultTarget = "***FAKE-ListAvailable***" -> listAvailable()
```

Comment:
If we add something like this we would add some "prefix" and prevent users from using it when defining targets. Honestly though I don't think we need to do this and there should be another way to do this (as there was previously)

Reply:
Agreed.  Will work on this :)